### DISCUSSION TITLE
Are these the same face?

### DISCUSSION THREAD

**Discussion by oledid** [immich-app/immich#7967] [General]
The competing app has a feature that you sometimes gets asked "are these the same face", which is a quick method to merge two face groups.

I have set my "same person" detection quite low, because of among other things: twins in the family. This works quite well, but I have quite a few faces I need to merge manually.

This could be done quicker if we had a separate task where the "same person detection value" where higher, and one could be suggested "are these two the same face" again and again with yes/no/unsure, one click merge if the answer was yes.

**Comment by Distearth** [comment]
I would like this as well. My daughter's first six months of photos are seperate from later ones. I gave them the same name but now I just have two people with the same name.
Thank you!

**Reply by oledid** [reply]
Hi

You can already easily fix this with the "Merge people" function. It might not be in the mobile version, but i know it is there in the web version. Easy peasy with only two face groups. What I am talking about here in this task is fixing it for hundreds or thousands of people.

**Reply by Distearth** [reply]
Ha! Thank you. I was doing all my matching in bed on my phone.

**Comment by radh21301** [comment]
I was looking for the exact same feature. I know this exists on google photos, but I also saw it on ente photos, and I realized it would be super useful to have in Immich.

<img width="399" height="851" alt="image" src="https://github.com/user-attachments/assets/2c30a905-6dfa-43cf-9dcb-47ce5d8d0fc7" />
